tests/unit/models/modelDebug.test.js

```diff
@@ -0,0 +1,29 @@
+const modelDebug = require('../../../api/model/modelDebug');
+const filledFields = [
+  { name: 'val1', value: 'Name' }
+];
+const blankFields = [
+  { name: 'val2', value: null },
+  { name: 'val3', value: NaN },
+  { name: 'val4', value: '' },
+  { name: 'val5', value: 0 },
+  { name: 'val6', value: false },
+];
+
+describe('checkMissingRequiredField', () => {
+  it('returns a required field if it is missing', () => {
+    expect(modelDebug.checkMissingRequiredField(filledFields.concat(blankFields))).toEqual(blankFields);
+  });
+  it('returns nothing if there are no missing fields', () => {
+    expect(modelDebug.checkMissingRequiredField(filledFields)).toEqual([]);
+  });
+});
+
+describe('renderMissingFieldDebugMessage', () => {
+  it('creates a debug message if a field is missing', () => {
+    expect(modelDebug.checkMissingRequiredField(filledFields.concat(blankFields))).toEqual(blankFields);
+  });
+  it('does not create a debug message if there are no missing fields', () => {
+    expect(modelDebug.checkMissingRequiredField(filledFields)).toEqual([]);
```

Comment:
According to https://github.com/NYPL/dgx-patron-creator-service/blob/development/api/model/modelDebug.js#L23, this should return an empty array.

I think the bigger problem for the tests for `renderMissingFieldDebugMessage` is that it's not checking to see if the `Missing ${element.name}` string was added to the returned object.

Reply:
Great catch @EdwinGuzman.  The test for `renderMissingFieldDebugMessage` was calling the wrong method, so I fixed that.